src/cmd.ts

```diff
@@ -319,13 +320,35 @@ async function cmdBuild(argv: BuildCmdArgvType) {
     });
   };
 
+  const buildRSSFeed = () => {
+    const rssPath = path.join(outputPath, "rss.xml");
+    const feed = new RSS({
+      title: appContext.title,
+      description: appContext.title,
+      feed_url: rssPath,
+      site_url: outputPath,
+      pubDate: new Date(),
+      ttl: 60,
+    });
+    pages.map(async (page) => {
```

Comment:
Nit but does this function need to be `async`?

Reply:
good call.